sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/availabilitystrategy/AvailabilityStrategy.java

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.cosmos.availabilitystrategy;
```

Comment:
Just my 2 cents: I think a separate package for this is too much - I would keep it in cosmos package (not model)

Reply:
Removed